EngineLayer/Silac/SilacConversions.cs

```diff
@@ -409,15 +409,16 @@ public static HashSet<FlashLFQ.ProteinGroup> CleanPastProteinQuant(HashSet<Flash
                 }
                 else //multiplex
                 {
-                    //foreach original file
```

Comment:
here's the bug fix: it's moving the loop order so that it's peptides first, then files. Previously, a peptide was being generated for each file, and then a crash downstream would happen because multiple entries existed for each file